### ISSUE TITLE
AkVCamManager command not found

### ISSUE THREAD

**Issue by kirk86** [webcamoid/akvirtualcamera#31]
## Summary

Just installed binary for macOS and followed the steps in the build wiki to for `lauchctl` in order to launch the service

## Current Behavior

In my process on one minute I see the AkVCAMAssistant and the other it's gone. 
Plus when I try the command `AkVCamManager` on the command line it's not found

## Expected Behavior

Shouldn't all command line commands be automatically recogniised after installation?
Plus, this should be working as stand-alone without webcamoid right?

## Steps to Reproduce
1. Install .pkg binary from latest stable realease
2. execute `sudo launchctl enable system/org.webcamoid.cmio.AkVCam.Assistant`
3. execute `sudo launchctl bootstrap system /Library/LaunchDaemons/org.webcamoid.cmio.AkVCam.Assistant.plist`
4. type `AkVCamManager` in CLI, not found return error

I think the real issue is with step 2 since executing it and double checking the running processes doesn't show up AkVCam.Assistant. Any ideas what's going on here?

Also, every time step 3 is executed I get the following message:
```
sudo launchctl bootstrap system /Library/LaunchDaemons/org.webcamoid.cmio.AkVCam.Assistant.plist
/Library/LaunchDaemons/org.webcamoid.cmio.AkVCam.Assistant.plist: service already bootstrapped
Bootstrap failed: 37: Operation already in progress
```

## Your Environment

* akvirtualcamera inormation: (9.1.0) (mac)
* Operating System information: (macOS) (11.5.1 and Big Sur) (intel)

**Comment by hipersayanX** [comment]
The manager must be called with the full path to it:

```
/Applications/AkVirtualCamera/AkVirtualCamera.plugin/Contents/Resources/AkVCamManager
```

If I remember well (I'm no in Mac right now).

**Comment by kirk86** [comment]
@hipersayanX thanks for the reply, in the `/Applications/AkVirtualCamera/AkVirtualCamera.plugin/Contents/Resources/` folder there were only two files an `uninstall.sh` and `filename.plist` whose name I don't recall, hence I used the `filename` placeholder. There was no AkVCamManager executable as far as I can remember.

**Comment by hipersayanX** [comment]
I think you are a bit confused, when you install the camera using the installer, the manager should be inside _/Applications/AkVirtualCamera/AkVirtualCamera.plugin/Contents/Resources/_.

> there were only two files an uninstall.sh and filename.plist

The _uninstall.sh_ file is inside _/Applications/AkVirtualCamera_, and there aren't such file called _filename.plist_, maybe you are confusing it with the Info.plist inside the _Contents_ folder.
Now, if you are compiling from  source code, the manager is located at the build folder, more or less _/akvirtualcamera-build-folder/build/AkVirtualCamera.plugin/Contents/Resources/_.

**Comment by kirk86** [comment]
> I think you are a bit confused, when you install the camera using the installer, the manager should be inside

you're right my confusion was this
![image](https://user-images.githubusercontent.com/2902390/144014836-77509b99-cded-44b0-8d35-790de0786972.png)

I couldn't find out initially were AkVCamManager manager was located, and double clicking the `AkVirtualCamera.plugin` didn't do much, hence my confusion

BTW I installed through the latest stable release `.pkg` container found here.


> there aren't such file called _filename.plist_, maybe you are confusing it with the Info.plist inside the _Contents_ folder.

yep, thanks as I explained I couldn't remember the `Info.plist` that's why I used `filename.plist` were `filename` was meant to be a placeholder for the `Info` which I couldn't remember.

 
This is the important bit of information that should be somewhere on the wiki

> The manager must be called with the full path to it:
> 
> ```
> /Applications/AkVirtualCamera/AkVirtualCamera.plugin/Contents/Resources/AkVCamManager
> ```

It seems to be working now, I'm closing this issue for the moment, if I get lost again I might ask another question, thanks!
